src/originalBudgetTrackingApp/index.ts

```diff
@@ -1,25 +1,16 @@
-import _ from 'lodash';
 import moment from 'moment';
-import * as bankScraper from './bankScraper';
-import { ScaperScrapingResult, Transaction } from './bankScraper';
-import * as ynab from './outputVendors/ynab/ynab';
-import { EnrichedTransaction } from './commonTypes';
-import * as categoryCalculation from './categoryCalculationScript';
+import { scrapeFinancialAccountsAndFetchTransactions } from '@/originalBudgetTrackingApp/import/importTransactions';
+import { createTransactionsInExternalVendors } from '@/originalBudgetTrackingApp/export/exportTransactions';
+import * as bankScraper from './import/bankScraper';
 import * as configManager from './configManager/configManager';
-import outputVendors from './outputVendors';
-import { Config } from './configManager/configManager';
+import outputVendors from './export/outputVendors';
+
+export { printYnabAccountData } from './setupHelpers';
 
 export { outputVendors };
 export { configManager };
 export const { inputVendors } = bankScraper;
 
-type AccountToScrapeConfig = configManager.AccountToScrapeConfig;
-
-const TRANSACTION_STATUS_COMPLETED = 'completed';
-const DATE_FORMAT = 'DD/MM/YYYY';
-
-export const { getYnabAccountDetails } = ynab;
-
 export async function scrapeAndUpdateOutputVendors() {
```

Comment:
OK, you can stay here, but if you're growth I will throw you to a different file!